site/docs/tools/client_config.md

```diff
@@ -107,35 +107,112 @@ environment-specific prefixes for brevity. Nonetheless, in practice the property
 given appropriate prefixes (as in the example above) for them to be recognized by the tools and Nessie
 code.
 
-The value of the `authentication.type` property can be one of the following:
+The value of the `nessie.authentication.type` property can be one of the following:
 
 * `NONE` (default)
 * `BEARER`
+* `OAUTH2`
 * `AWS`
 * `BASIC` (deprecated)
 
 ## Authentication Type `NONE`
 
-For the Authentication Type `NONE` only the `authentication.type` property needs to be set.
+For the Authentication Type `NONE` only the `nessie.authentication.type` property needs to be set.
```

Comment:
Just wondering: why do we need the `nessie` prefix here?

Reply:
Well I just looked at `NessieConfigConstants`: all constants start with `nessie.` so I thought the docs were wrong. Are we supposed to remove that prefix when running the client?

Reply:
Yeah, it can be misleading, I guess :thinking: IIRC, this doc applies to Spark/Iceberg catalog configuration where the `nessie.` prefix is replaced with the actual catalog name (there could be multiple names).

Cf. https://github.com/apache/iceberg/blob/522480dc4dff7305808403281980f6afdedf9cf1/nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java#L86

Reply:
I tend to think that presenting an Iceberg-centric picture here is preferable because the doc page is about "tools" so it should hopefully be easier for end users to understand.

Reply:
Example for Spark:
```
bin/spark-shell [...] --conf spark.sql.catalog.mycatalog.catalog-impl=org.apache.iceberg.nessie.NessieCatalog --conf spark.sql.catalog.mycatalog.uri=$NESSIE_URI  --conf spark.sql.catalog.mycatalog.authentication.type=BEARER --conf spark.sql.catalog.mycatalog.authentication.token=$NESSIE_TOKEN
```

Reply:
I see, thanks for the explanation. It's really confusing for newcomers like me especially because at the top of the same page the options have the `nessie.` prefix:

https://github.com/projectnessie/nessie/blob/nessie-0.57.0/site/docs/tools/client_config.md?plain=1#L11

But OK, I will revert the prefix changes.

Reply:
Good point. We have this in the `Authentication Settings` section:
```
The property names are shown without environment-specific prefixes for brevity. Nonetheless, in practice the property names should be given appropriate prefixes (as in the example above) for them to be recognized by the tools and Nessie code.
```
.. maybe we could have another clarifying statement around the top-level table too... WDYT?

Reply:
Hmm I think that's enough in fact. I should have read the document more carefully.